simplesamlphp/simplesamlphp/CVE-2017-12868.yaml

```diff
@@ -0,0 +1,8 @@
+title:     Session fixation and authentication bypass (authcrypt module)
+link:      https://simplesamlphp.org/security/201705-01
+cve:       CVE-2017-12868
+branches:
+    1.14.x:
+        time:     2017-05-05 10:46:00
+        versions: ['>=1.14.12', '<1.14.14']
```

Comment:
this looks wrong. The link given just above says that *all* versions up to 1.14.13 are impacted, not only 1.4.12 and 1.4.13.